Makefile.am

```diff
@@ -128,7 +128,11 @@ EXTRA_DIST = test/fixtures/empty_file \
 
 TESTS = test/run-tests
 check_PROGRAMS = test/run-tests
+if OS390
+test_run_tests_CFLAGS =
```

Comment:
Forgot to also mention that the zos compiler does not recognize -W option format.
So the compile fails with "option not recognized"